src/main/cpp/src/ftos_converter.cu

```diff
@@ -0,0 +1,1197 @@
+// Copyright 2018 Ulf Adams
+//
+// The contents of this file may be used under the terms of the Apache License,
+// Version 2.0.
+//
+//    (See accompanying file LICENSE-Apache or copy at
+//     http://www.apache.org/licenses/LICENSE-2.0)
+//
+// Alternatively, the contents of this file may be used under the terms of
+// the Boost Software License, Version 1.0.
+//    (See accompanying file LICENSE-Boost or copy at
+//     https://www.boost.org/LICENSE_1_0.txt)
+//
+// Unless required by applicable law or agreed to in writing, this software
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.
+
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
```suggestion
/*
 * Copyright 2018 Ulf Adams
 * Copyright (c) 2023, NVIDIA CORPORATION.
```

We are not licensing the file via Boost, only via ASLv2.  There is no LICENSE-Apache or LICENSE-Boost files in this repo as were referenced in the original text, nor do we want to add them.  We are choosing to license via ASLv2.  We need to preserve the copyright and attribute in NOTICE, which has been done.